homeassistant/components/wallbox/number.py

```diff
@@ -23,20 +45,27 @@ async def async_setup_entry(hass, config, async_add_entities):
     except InvalidAuth:
         pass
```

Comment:
```suggestion
        return
```
Then we can remove the `else:` and outdent below.

Reply:
I don't think that works, the idea is here that we only want to add the number component if we don't have an InvalidAuth exception. (Functionally, if somebody is not authorized to do this -> don't add it.) So, if I put the `add_entities` outside of the else block it will always run, right? (I think replacing the pass with return works however, just not the removing else part.)

Reply:
It will work. If we return when there's an exception the rest of the function body will not run.

Reply:
Of course, it returns from the entire function not just the try/except. I understand now, changed it.